doc/man1/openssl.pod

```diff
@@ -77,161 +77,161 @@ B<list>, or B<no->I<XXX> itself.)
 
 =over 4
 
-=item L<B<asn1parse>|asn1parse(1)>
+=item L<asn1parse(1)>
```

Comment:
I would prefer this:

```
=item B<asn1parse>
```

(and not just for this one)

Reply:
But we want the '(1)' to refer to the man1 section.  Look at all the other ```L<>``` references, such as the SEE ALSO sections.

Reply:
What about the message digest and the encoding and cipher commands?

The SEE ALSO section is different, you'd expect links there. 

I think these were formatted like they were so the itemized subcommands would have direct links to their pages in html, but would otherwise look like subcommands, not like manpage references.

Reply:
I insist, FYI

Reply:
And, in retrospect, you were right to do so.  Additional commit pushed.  (all will be squashed)